editor/js/libs/app/package.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "three-app",
+	"version": "1.0.0",
+	"private": true,
+	"scripts": {
+		"dev": "vite",
+		"build": "vite build",
+		"preview": "vite preview"
+	},
+	"dependencies": {
+		"three": "github:mrdoob/three.js#dev",
```

Comment:
Can we use latest `prod` release here? 

Most published apps will be produced from `https://threejs.org/editor` so the version of `three` in the published app should match the current version of the editor in production?

Reply:
Or in other words: If an app gets published with `r166`, the `three` dependency should hold the value `~0.166.0`. If the app gets published one month later again with `r167`, the new `package.json` should hold `~0.167.0`.

Reply:
fixed.